umurmur/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM dxrto/base
+LABEL authors="krageon <krageon@gmail.com>"
+RUN xbps-install -Sy umurmur
```

```suggestion
RUN xbps-install -Sy umurmur && xbps-remove -Oo
```
So it removes the cached packages :)

---

I patched it in